src/types/query-types.ts

```diff
@@ -49,4 +49,9 @@ export type RangeCriterion = {
    * Inclusive end date-time.
    */
   to?: string;
+};
+
+export type PaginationCursor = {
+  itemId: string;
```

Comment:
Any reason why this cannot be named `messageCid`?

Reply:
Yup! And I went ahead and just removed references to `itemId` as they don't really make sense anymore.